README.md

```diff
@@ -33,33 +33,50 @@ But there's more.
 You can cache results by configuring the resolver to use a `CachedExecutor`:
 
 ```php
-    $loop = React\EventLoop\Factory::create();
-    $factory = new React\Dns\Resolver\Factory();
-    $dns = $factory->createCached('8.8.8.8', $loop);
+$loop = React\EventLoop\Factory::create();
+$factory = new React\Dns\Resolver\Factory();
+$dns = $factory->createCached('8.8.8.8', $loop);
+
+$dns->resolve('igor.io')->then(function ($ip) {
+    echo "Host: $ip\n";
+});
 
-    $dns->resolve('igor.io')->then(function ($ip) {
-        echo "Host: $ip\n";
-    });
+...
 
-    ...
+$dns->resolve('igor.io')->then(function ($ip) {
+    echo "Host: $ip\n";
+});
 
-    $dns->resolve('igor.io')->then(function ($ip) {
-        echo "Host: $ip\n";
-    });
-    
-    $loop->run();
+$loop->run();
 ```
 
 If the first call returns before the second, only one query will be executed.
 The second result will be served from cache.
 
-## Todo
+## Install
+
+The recommended way to install this library is [through Composer](http://getcomposer.org).
+[New to Composer?](http://getcomposer.org/doc/00-intro.md)
+
+This will install the latest supported version:
+
+```bash
+$ composer require react/dns:~0.4.0
```

Comment:
Regarding "composer require react/dns:~0.4.0" vs "composer require react/dns":
The former installs the "latest supported version" (i.e. the series of releases that still receive support) whereas the latter installs the latest compatible version.
I understand your concerns that this version number would need to be updated, but IMHO this is a good thing in this case.

Personally, I don't have strong opinion on this tbh, so I'm good with it either way.